src/internal.c

```diff
@@ -26216,6 +26223,14 @@ static int HashSkeData(WOLFSSL* ssl, enum wc_HashType hashType,
          ssl->buffers.digest.length = (unsigned int)digest_sz;
 
         /* buffer for hash */
+        if(!ssl->buffers.digest.buffer) {
```

Comment:
Minor nitpicking, please adjust the spacing between if and (. We usually do `if (` instead of `if(`. Also later on match the 'var = 0' instead of 'var=0'.

Reply:
Updated spacings